assets/js/theme/common/carousel/utils/tooltipSetup.js

```diff
@@ -17,7 +16,7 @@ const setupArrowTooltips = (...arrowNodes) => {
 };
 
 const setupDotTooltips = ($dots) => {
-    $dots.children().each((idx, dot) => setupTooltipAriaLabel($(dot).find('button')));
+    $dots.children().each(() => setupTooltipAriaLabel($('.js-carousel-dot')));
```

Comment:
Is it works correctly? As I can see now we passing collection of all dots to setupTooltipAriaLabel on each loop iteration, but I believe we should pass one dot node on each iteration

Reply:
done